docs/content/latest/yugabyte-platform/security/enable-encryption-at-rest.md

```diff
@@ -60,15 +60,15 @@ You can verify that encryption at rest has been successfully configured as follo
+If your configuration includes Equinix SmartKey, the following occurs: once the universe has started being created, a security object will be created with SmartKey through REST API with the universe UUID as the name of the security object. This object will then be exported to Yugabyte Platform as the universe key. Once received, Yugabyte Platform sends the key to all masters to persist. Yugabyte Platform does not persist any information about the security object beyond a reference to the object `kId`. At this point, data within the universe will be encrypted at rest using data keys encrypted with the aforementioned universe key generated through the selected KMS configuration. For more information, see [Create a KMS configuration using Equinix SmartKey](../create-kms-config/equinix-smartkey/). You can access the **Security Object** page for the Equinix SmartKey account that was used to create the KMS configuration. A security object should exist with a name matching the universe UUID that was configured to use encryption at rest with the SMARTKEY configuration (defaults to AES 256).
 
-If your configuration includes AWS KMS, the following occurs: once the universe has been created with encryption at rest enabled, Yugabyte Platform persists the ciphertext of the universe key (because AWS does not persist any CMK-generated data keys themselves) and requests the plaintext of the universe key from AWS KMS using the KMS configuration whenever it needs to provide the universe key to the master nodes. For more information, see [Create a KMS configuration using AWS KMS](../create-kms-config.md/aws-kms/).
+If your configuration includes AWS KMS, the following occurs: once the universe has been created with encryption at rest enabled, Yugabyte Platform persists the ciphertext of the universe key (because AWS does not persist any CMK-generated data keys themselves) and requests the plaintext of the universe key from AWS KMS using the KMS configuration whenever it needs to provide the universe key to the master nodes. For more information, see [Create a KMS configuration using AWS KMS](../create-kms-config/aws-kms/).
```

Comment:
Anything need to be added if config includes Hashicorp?

Reply:
Good question. Let me check with the developer